app/index/controller.js

```diff
@@ -86,16 +100,33 @@ export default class IndexController extends Controller {
    * @private
    */
   _activitySum() {
+    // Do not trigger updates whne there is no active activity, but let it run once to
+    // null the duration.
+    if (
+      !this.tracking.hasActiveActivity &&
+      this._activeActivityDuration.get("seconds") === 0
+    ) {
+      return;
+    }
     const duration = this._activities
       .filterBy("active")
       .reduce((total, current) => {
         return total.add(moment().diff(current.get("from")));
       }, moment.duration());
 
     this._activeActivityDuration = duration;
+
+    // Save latest activitySum for display while reports are generated.
+    // See activitySum getter.
+    scheduleOnce("afterRender", this, "_storeLastActivitySum");
+
     return duration;
   }
 
+  _storeLastActivitySum() {
+    this._lastActivitySum = this.activitySum;
+  }
+
```

Comment:
My brain tells me there would probably be a better way to solve this but sadly lacks the power to tell me the solution.

Reply:
Mine got exhausted by this code as well... this whole mechanism is weird. I just tried to get it running again.